src/pages/Graph/SummaryPanelCommon.tsx

```diff
@@ -11,13 +11,18 @@ import { Response } from '../../services/Api';
 import { serverConfig } from '../../config/ServerConfig';
 import { decoratedNodeData } from 'components/CytoscapeGraph/CytoscapeGraphUtils';
 import { Badge } from '@patternfly/react-core';
+import { PfColors } from 'components/Pf/PfColors';
 
 export enum NodeMetricType {
   APP = 1,
   WORKLOAD = 2,
   SERVICE = 3
 }
 
+export const summaryHeader: React.CSSProperties = {
+  backgroundColor: PfColors.White
+};
```

Comment:
We generally prefer [typestyle](https://github.com/typestyle/typestyle) over the inline css that this generates. see https://github.com/kiali/kiali-ui/blob/master/src/components/CytoscapeGraph/graphs/GraphStyles.ts#L65

Reply:
+1 and I tried that but could not figure out how to combine a static className with a typestyle definition.  Feel free to make that change if you know how to do it, but I'm going to merge now that we have QE approval.